run/pubsub/go.mod

```diff
@@ -0,0 +1,3 @@
+module run_pubsub
```

Comment:
Probably use the default module name (with the full `golang-samples` path)? That's what we've done for others. Not clear what the best way to handle nested modules in golang-samples is.